cpg-core/src/main/java/de/fraunhofer/aisec/cpg/frontends/cpp/CLanguage.kt

```diff
@@ -26,20 +26,27 @@
 package de.fraunhofer.aisec.cpg.frontends.cpp
 
 import de.fraunhofer.aisec.cpg.TranslationConfiguration
-import de.fraunhofer.aisec.cpg.frontends.HasFunctionPointers
-import de.fraunhofer.aisec.cpg.frontends.HasStructs
-import de.fraunhofer.aisec.cpg.frontends.Language
+import de.fraunhofer.aisec.cpg.frontends.*
 import de.fraunhofer.aisec.cpg.passes.scopes.ScopeManager
 
 /** The C language. */
-class CLanguage : Language<CXXLanguageFrontend>(), HasStructs, HasFunctionPointers {
+class CLanguage :
+    Language<CXXLanguageFrontend>(),
+    HasStructs,
+    HasFunctionPointers,
+    HasQualifier,
+    HasElaboratedTypeSpecifier {
     // TODO: Shouldn't there also be the "h" ending?
     override val fileExtensions: List<String>
-        get() = listOf("c")
+        get() = listOf("c", "h")
```

Comment:
I thought we excluded header files on purpose because otherwise they will get parsed twice if they are also included in the C files?

Reply:
Hmmm, I don't know about that but one test failed when it wasn't included because it didn't know which frontend was responsible for the header file.

Reply:
Looks like the old frontend also included it.